protocol/x/epochs/module.go

```diff
@@ -145,7 +147,8 @@ func (AppModule) ConsensusVersion() uint64 { return 1 }
 
 // BeginBlock executes all ABCI BeginBlock logic respective to the epochs module.
 func (am AppModule) BeginBlock(ctx context.Context) error {
+	sdkCtx := lib.UnwrapSDKContext(ctx, types.ModuleName)
```

Comment:
nit: move this to below the latency metric

Reply:
Will do in second pr, got another one coming up